qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -131,29 +207,35 @@ def __init__(
             link_parameters: if set to True then the amplitude and DRAG parameters of the
                 X and Y gates will be linked as well as those of the SX and SY gates.
         """
-        super().__init__(basis_gates, default_values)
         self._link_parameters = link_parameters
+        self._use_drag = use_drag
```

Comment:
Question about class design: what is the criteria of flag vs defining new library? If target is two-level system, probably we want to set `use_drag=False`. Then, this is no longer called `*Transmon`. I think for transmon devices we don't need to set this flag and we can always give finite drag parameter. I prefer to have separate `TwoLevelSystem` and `Transmon` libraries without and with drag parameter. Former can be used for unittest or educational purpose (simulator) in my case.

Reply:
I have come across cases where transmons have been very weakly driven with long pulses. In such a setting Drag is not needed. We could have a separate library for Gaussian only but I don't like the idea of having an extra class for a single setting that can be solved with a boolean flag. Once we start adding CR gates to the library this might change and we can rethink how to best define libraries. However, this goes beyond the scope of this PR which is focused on serialization.

Reply:
Yes, I agree. This is not the scope of the PR but I also don't prefer flagging. If the library takes a flag it somewhat looks like a factory.

Reply:
Resolving as out of scope.